cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java

```diff
@@ -104,18 +104,22 @@ protected String caName() {
      *
      * @param namespace                             Namespace of the Kafka cluster
      * @param clusterName                           Name of the Kafka cluster
-     * @param existingSecret                        Existing Secret with the existing certificates (or null if it does not exist yet)
+     * @param existingCertificates                  Existing certificates (or null if they do not exist yet)
+     * @param nodes                                 Nodes that are part of the Cruise Control cluster
      * @param isMaintenanceTimeWindowsSatisfied     Flag indicating whether we can do maintenance tasks or not
+     * @param forceRenew                            Flag indicating whether to renew the existing certificates
      *
-     * @return  Map with CertAndKey object containing the public and private key
+     * @return Map with CertAndKey object containing the public and private key
      *
-     * @throws IOException  IOException is thrown when it is raised while working with the certificates
+     * @throws IOException IOException is thrown when it is raised while working with the certificates
      */
     protected Map<String, CertAndKey> generateCcCerts(
             String namespace,
             String clusterName,
-            Secret existingSecret,
-            boolean isMaintenanceTimeWindowsSatisfied
+            Map<String, CertAndKey> existingCertificates,
+            Set<NodeRef> nodes,
+            boolean isMaintenanceTimeWindowsSatisfied,
+            boolean forceRenew
```

Comment:
Here - as well as in the other methods below - you call it `forceRenewe`. But you pass into it the value `hasCaCertGenerationChanged`. Given force-renew is an annotation for force certificate renewal, I think we should rename this to better correspond to what indicates here? E.g. `caCertGenerationChanged`?

Reply:
Yeah I see your point, I've renamed it